src/app/cases/creator/case-creator-submit.component.spec.ts

```diff
@@ -282,5 +284,165 @@ describe('CaseCreatorSubmitComponent', () => {
     expect(draftService.createOrUpdateDraft).toHaveBeenCalledWith(JID, CTID, EVENT_TRIGGER.case_id, SANITISED_EDIT_FORM);
     expect(router.navigate).toHaveBeenCalledWith([`case/${JID}/${CTID}/${EVENT_TRIGGER.case_id}`]);
   });
+});
+
+describe('CaseCreatorSubmitComponent with Save and Resume enabled', () => {
```

Comment:
This should end ...disabled instead of ...enabled but let's not raise it here. I will fix it in next PR 3064